pkg/cni/version.go

```diff
@@ -53,8 +53,7 @@ var (
 			"v1.12",
 			// NOTE: as of 1.13.0, we moved to Application infra for Cilium CNI management and started using real smever
 			// See pkg/cni/cilium docs for details on introducing a new version.
-			"1.13.0",
-			"1.13.3",
```

Comment:
Can we keep offering this version as customers can potentially configure IPSec? Since we default to 1.13.4,  I think it's fine to keep this one, but deprecating 1.13.0 is a good idea.